scss/_toasts.scss

```diff
@@ -47,4 +47,5 @@
 
 .toast-body {
   padding: $toast-padding-x; // apply to both vertical and horizontal
+  word-break: break-all;
```

Comment:
Cards have `word-wrap: break-word;`—should we use that instead, @twbs/css-review?

```suggestion
  word-wrap: break-word;
```

Reply:
Indeed, `word-wrap` is better. As explained on [MDN `overflow-wrap` page](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap):

> **Note**: In contrast to `word-break`, `overflow-wrap` will only create a break if an entire word cannot be placed on its own line without overflowing.

`overflow-wrap` being the new name of `word-wrap`, the latter still working as an alias.

Reply:
BTW, since we dropped Edge Legacy, we could introduce (again) `overflow-wrap`. However we've been back and forth with it, so we might also wait for v6.